### PR TITLE
Story6 new sets of numbers

### PR DESCRIPTION
**Pull request by alexjames4**
Story 6 complete. ready for peer review.

CSS finessing and definition of done yet to be completed.

### CODE REVIEW

File: src/.DS_Store
Comment (alexjames4) [inline src/.DS_Store:1]: delete this

File: src/generateNumberCards.js [hunk 1]
```diff
@@ -1,19 +1,43 @@
...
 function generate3To5NumberCards() {
   for (let i = 0; i < generateRandomInt(3, 5); i++) {
-    let randomNumber = generateRandomInt(0, 100);
+    let randomNumber = generateRandomInt(0, 100)
     while (randomNumbersHolder.includes(randomNumber)) {
-      randomNumber = generateRandomInt(0, 100);
+      randomNumber = generateRandomInt(0, 100)
     }
-    randomNumbersHolder[i] = randomNumber;
+    randomNumbersHolder[i] = randomNumber
```
Comment (MrTKetchup) [inline src/generateNumberCards.js:15]: change this to .push

File: style.css [hunk 2]
```diff
@@ -53,6 +77,7 @@ body {
   padding-left: 60px;
   max-width: fit-content;
 }
+
```
Comment (alexjames4) [inline style.css]: lines 76 - 79 not needed as no p tags in the cards anymore

File: index.html [hunk 3]
```diff
@@ -35,6 +34,7 @@ <h2>Instructions</h2>
           <button class="start-btn">Begin Caterpillar Countdown</button>
         </a>
       </div>
+      <img src="src/menu-caterpillar-image.png" class="caterpillar-image" id="caterpillar-right"/>
```
Comment (CMeier556) [inline index.html]: alternative text for images

File: style.css [hunk 4]
```diff
@@ -3,34 +3,58 @@
...
   padding: 10px;
   display: flex;
   justify-content: center;
 }
+
 body {
   background-color: #568203;
 }
+
+.menu-screen-container {
```
Comment (mporam) [inline style.css:21]: the content in this isnt centered, looks weird

File: src/generateNumberCards.js [hunk 5]
```diff
@@ -1,19 +1,43 @@
...
     document.querySelector(
       ".number-cards-container"
-    ).innerHTML += `<div class="number-card">${randomNumber}</div>`;
+    ).innerHTML += `<div class="number-card">${randomNumber}</div>`
   }
+  randomNumbersInSequence = randomNumbersHolder.sort((a, b) => a - b)
+
+  const cards = document.querySelectorAll(".number-card")
+  cards.forEach((card) => {
```
Comment (mporam) [inline src/generateNumberCards.js]: no brackets round `card`

File: src/generateNumberCards.js [hunk 6]
```diff
@@ -1,19 +1,43 @@
...
+    card.addEventListener("click", (e) => {
```
Comment (mporam) [inline src/generateNumberCards.js]: no brackets round `e`

File: src/highlight.js [hunk 7]
```diff
@@ -1,9 +1,6 @@
-const cards = document.querySelectorAll(".number-card")
-let randomNumbersInSequence = randomNumbersHolder.sort(((a, b) => a-b))
-let correctNumbersHolder = []
-
 function removeAllCards() {
   document.querySelector('.number-cards-container').innerHTML = ""
+  correctNumbersHolder = []
 }
 
 function removeMinimumNumber() {
```
Comment (mporam) [inline src/highlight.js:6]: the shift method has no properties

File: src/generateNumberCards.js [hunk 8]
```diff
@@ -1,19 +1,43 @@
...
+      if (e.target.textContent == randomNumbersInSequence[0]) {
+        e.target.classList.add("highlighted-number-card")
+        removeMinimumNumber()
+        if (randomNumbersInSequence.length == 0) {
```
Comment (mporam) [inline src/generateNumberCards.js]: use `===` when possible

File: src/generateNumberCards.js [hunk 9]
```diff
@@ -1,19 +1,43 @@
...
+          removeAllCards()
+          add1toScore()
+          generate3To5NumberCards()
+        }
+      } else {
+        resetNumbersInSequence()
+        cards.forEach((card) => {
```
Comment (mporam) [inline src/generateNumberCards.js]: remove brackets from `card`

File: src/score-and-timer.js [hunk 10]
```diff
@@ -1,12 +1,16 @@
+count = 0
```
Comment (mporam) [inline src/score-and-timer.js]: no `let`?

File: src/score-and-timer.js [hunk 11]
```diff
@@ -1,12 +1,16 @@
...
+
+function add1toScore() {
+    count++
+    document.querySelector('.score span').textContent = count
+}
+
 const timer = timeLeft => {
     document.querySelector('.timer').textContent = 'Time Left: ' + timeLeft + ' seconds'
```
Comment (mporam) [inline src/score-and-timer.js]: switch for ` to be consistent with rest of code

File: style.css [hunk 12]
```diff
@@ -49,10 +73,10 @@ body {
   background-color: #686868;
 }
 
-.number-card p {
-  padding-left: 60px;
-  max-width: fit-content;
+.number-card {
```
Comment (mporam) [inline style.css]: you already have this selector on line 65. which is also missing a line break before it

File: src/generateNumberCards.js [hunk 13]
```diff
@@ -1,19 +1,43 @@
...
+          card.classList.remove("highlighted-number-card")
```
Comment (mporam) [inline src/generateNumberCards.js]: picky but im not a fan of this class name. could just be `highlighted` or `active` would be common. if you shorten this class name this code could then become:
```
cards.forEach(card => card.classList.remove('active'))
```
which is much tidier

**Review by mporam — CHANGES_REQUESTED** [review]

File: style.css (hunk 4 above)
Comment (alexjames4) [reply style.css:21]: Yep, we will be tweaking shortly

Comment (alexjames4) [reply style.css:21]: tweaks made

**Review by mporam — APPROVED** [review]

**Pull request merged**